aea/cli/add.py

```diff
@@ -77,6 +78,12 @@ def _find_connection_locally(ctx, connection_name):
         logger.error(str(e))
         sys.exit(1)
 
```

Comment:
Note, we need to recursively load protocol dependencies and therefore also need a protocol dependency list in the connection.yaml.